packages/webapp/src/routes/credits-purchase/cdd-cya.js

```diff
@@ -0,0 +1,48 @@
+import creditsPurchaseConstants from '../../utils/credits-purchase-constants.js'
+import { dateToString } from '../../utils/helpers.js'
+
+const getApplicationDetails = (session) => {
+  const nationality = session.get(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_NATIONALITY)
+  const nationalityHtml = nationality ? Object.values(nationality).filter(n => n !== '').join('<br/>') : ''
+
+  return {
+    dueDiligence: {
+      individualOrOrg: session.get(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_USER_TYPE),
+      individualOrOrgUrl: creditsPurchaseConstants.routes.CREDITS_PURCHASE_INDIVIDUAL_OR_ORG,
+      checkDefraAccount: session.get(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_DEFRA_ACCOUNT_DETAILS_CONFIRMED) && 'Confirmed',
+      checkDefraAccountUrl: creditsPurchaseConstants.routes.CREDITS_PURCHASE_CHECK_DEFRA_ACCOUNT_DETAILS,
+      middleName: session.get(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_MIDDLE_NAME)?.middleName,
```

Comment:
is this right? CREDITS_PURCHASE_MIDDLE_NAME is an object? not a value?

Reply:
Yes it is - https://github.com/DEFRA/biodiversity-net-gain-service/blob/e7725f2e04239df262fb5e742304ae02122c85cc/packages/webapp/src/routes/credits-purchase/middle-name.js#L32